libs/developers-tx-result-adapter-spring-support/src/main/resources/META-INF/spring.factories

```diff
@@ -1,2 +1,3 @@
 org.springframework.boot.autoconfigure.EnableAutoConfiguration=\
- com.linecorp.link.developers.txresult.TxResultAutoConfiguration
+ com.linecorp.link.developers.txresult.TxResultAutoConfiguration, \
+ com.linecorp.link.developers.client.WebClientConfiguration
```

Comment:
maybe it caused bean duplication.

Reply:
Fixed at 562df0a3102c18d77b0e219cb0c68386fa716b9b